aioresponses/core.py

```diff
@@ -240,12 +270,15 @@ def add(self, url: 'Union[URL, str]', method: str = hdrs.METH_GET,
             repeat=repeat,
             timeout=timeout,
             reason=reason,
+            callback=callback,
         ))
 
-    async def match(self, method: str, url: URL) -> Optional['ClientResponse']:
+    async def match(
+            self, method: str, url: URL, *args: Tuple, **kwargs: Dict
```

Comment:
I don"t think `*args` are usefull. From what I checked `aiohttp` also skips any `*args`

Reply:
Agree, removed it.